packages/magiclink-auth-service-api/src/routes/api-docs.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const swaggerUi = require('swagger-ui-express');
+
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+const logger = require('../util/logger');
+
+const router = express.Router();
+
+let spec;
+
+try {
```

Comment:
I never knew you could state the functionality without declaring a function like this. Thank you.

Reply:
This implementation is based on the code that I found in another existing service. :)